Terminal.Gui/Windows/Wizard.cs

```diff
@@ -316,7 +316,11 @@ public override void Remove (View view)
 
 				SetNeedsDisplay ();
 				var touched = view.Frame;
-				contentView.Remove (view);
+				if (view == contentView || view.GetType().Name == "ContentView") {
```

Comment:
I'm LOLing here. 

In case anyone is confused, it's hacks like this that motivated me to implement the Adornments (`Frames`) concept in v2. 

;-)

Reply:
Well without the v1, for sure there will not exist a v2. But I understand you :-)